<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMainWindow</class>
 <widget class="QMainWindow" name="MyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>663</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sync-my-LÂ²P</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/Icons/Icons/magnifier.png</normaloff>:/Icons/Icons/magnifier.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="Browser" name="browserTab">
       <attribute name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/Icons/Icons/1419140450_678077-computer-128.png</normaloff>:/Icons/Icons/1419140450_678077-computer-128.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Dateibrowser</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
      <widget class="Options" name="optionsTab">
       <attribute name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/Icons/Icons/1419140515_678102-cog-128.png</normaloff>:/Icons/Icons/1419140515_678102-cog-128.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Einstellungen</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
      <widget class="QWidget" name="logTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/Icons/Icons/1419123220_edit-paste.png</normaloff>:/Icons/Icons/1419123220_edit-paste.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="logListWidget">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="isWrapping" stdset="0">
           <bool>false</bool>
          </property>
          <property name="modelColumn">
           <number>0</number>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Browser</class>
   <extends>QWidget</extends>
   <header>browser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Options</class>
   <extends>QWidget</extends>
   <header>options.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
